packages/sveltekit/README.md

```diff
@@ -150,20 +150,20 @@ The Sentry SvelteKit SDK mostly relies on [SvelteKit Hooks](https://kit.svelte.d
 
 ### 5. Vite Setup
 
-1. Add our `sentrySvelteKitPlugin` to your `vite.config.(js|ts)` file so that the Sentry SDK can apply build-time features.
-  Make sure that it is added before the `sveltekit` plugin:
+1. Add our `sentryVite` plugins to your `vite.config.(js|ts)` file so that the Sentry SDK can apply build-time features.
+   Make sure that it is added before the `sveltekit` plugin:
 
    ```javascript
     import { sveltekit } from '@sveltejs/kit/vite';
-    import { sentrySvelteKitPlugin } from '@sentry/sveltekit';
+    import { sentryVite } from '@sentry/sveltekit';
 
     export default {
-      plugins: [sentrySvelteKitPlugin(), sveltekit()],
+      plugins: [sentryVite(), sveltekit()],
```

Comment:
As discussed: I would lean towards calling this plugin something like `sentrySveltekit()` just to avoid ambiguity with the existing `@sentry/vite-plugin`. I'd rather have non-ambiguous plugins at the cost of more verbose names. The no. 1 reason for me is people googling stuff and multiple very similar sounding things coming up.